src/Bridge/Symfony/Bundle/Resources/config/graphql-ui.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
```

Comment:
This should be reverted. This action isn't here for GraphiQL in the first place. It is the main GraphQL entrypoint.